fix(iotdb): use the `ping` API to check the status of the IOTDB connections (#12602)
emqx/emqx: changes/ee/feat-12602.en.md

> Enhanced health checking for IOTDB connector, using its `ping` API replacing the old method which only validating via a socket connection.

Reviewer: ```suggestion
Enhanced health checking for IoTDB connector, using its `ping` API instead of just checking for an existing socket connection.
```
